instrumentation/jmx-metrics/src/main/resources/jmx/rules/jvm.yaml

```diff
@@ -81,15 +81,46 @@ rules:
       ThreadCount:
         metric: count
         desc: Number of threads
-  - bean: java.lang:type=OperatingSystem
+      DaemonThreadCount:
+        metric: daemon
```

Comment:
Should we make it `jvm.threads.daemon.count` or `jvm.daemon_threads.count` to make the semantic meaning clearer?